python/tvm/relay/op/transform.py

```diff
@@ -1823,3 +1823,40 @@ def invert_permutation(data):
         relay.invert_permutation(data) = [2, 4, 3, 0, 1]
     """
     return _make.invert_permutation(data)
+
+
+def stft(data, n_fft, hop_length, win_length, window, normalized, onesided):
+    """
+    The STFT computes the Fourier transform of short overlapping windows of the input.
+    This giving frequency components of the signal as they change over time.
```

Comment:
giving -> gives

Fix the same typo in other files too.

Reply:
Fixed.